resources/fetchersManager/data/data.go

```diff
@@ -38,13 +39,13 @@ type Data struct {
 	// Duration between two consecutive cycles
 	interval time.Duration
 
-	fetchers FetchersRegistry
+	fetchers registry.FetchersRegistry
 
 	ctx    context.Context
 	cancel context.CancelFunc
 }
 
-func NewData(ctx context.Context, log *logp.Logger, interval time.Duration, timeout time.Duration, fetchers FetchersRegistry) (*Data, error) {
+func NewData(ctx context.Context, log *logp.Logger, interval time.Duration, timeout time.Duration, fetchers registry.FetchersRegistry) (*Data, error) {
```

Comment:
Given the opportunity by the refactoring, maybe we should rename this field to something more descriptive? This struct seems to be a wrapper around fetchers

Reply:
changed to `Manager`